src/commands/MigrationCreateCommand.ts

```diff
@@ -73,14 +73,16 @@ export class MigrationCreateCommand {
      * Gets contents of the migration file.
      */
     protected static getTemplate(name: string, timestamp: number): string {
-        return `import {MigrationInterface, QueryRunner, Connection} from "typeorm";
+        return `import {Connection, EntityManager, MigrationInterface, QueryRunner} from "typeorm";
 
 export class ${name}${timestamp} implements MigrationInterface {
 
-    async up(queryRunner: QueryRunner, connection: Connection, entityManager?: EntityManager): Promise<any> {
+    public async up(queryRunner: QueryRunner, connection: Connection, entityManager?: EntityManager): Promise<any> {
+        ;
```

Comment:
We have not been setting this rule explicitly, it is part of the default rule set. In any case, would you still consider this pull request? Or do you need me to change anything?